src/rpc/net.cpp

```diff
@@ -83,6 +83,7 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
             "    \"addr\":\"host:port\",      (string) The IP address and port of the peer\n"
             "    \"addrbind\":\"ip:port\",    (string) Bind address of the connection to the peer\n"
             "    \"addrlocal\":\"ip:port\",   (string) Local address as reported by the peer\n"
+            "    \"mapped_as\":\"mapped_as\", (string) The AS in the BGP route to the peer used for diversifying peer selection\n"
```

Comment:
The `mapped_as` value is currently returned as an integer.

Reply:
Will address this in #17812.

Reply:
Doesn't returning it as an integer make sense, as it is an integer? So I think we should change the documentation.

Reply:
> Doesn't returning it as an integer make sense, as it is an integer? So I think we should change the documentation.

Thanks! Agreed.